Table 1. Bacterial strains
<table border=1>
<tr>
<td>Strain</td>
<td>Relevant characteristics</td>
<td>Source or reference</td>
</tr>
<tr>
<td>W3110</td>
<td>wild type</td>
<td>Laboratory stock</td>
</tr>
<tr>
<td>JE7951</td>
<td>spr-1</td>
<td>$(7)$</td>
</tr>
<tr>
<td>JE7934</td>
<td>prC $\Delta:$ neo</td>
<td>$(7)$</td>
</tr>
<tr>
<td>JE7456</td>
<td>str xyl argG</td>
<td>Laboratory stock</td>
</tr>
<tr>
<td>CAG12072</td>
<td>sqj-203::Th10</td>
<td>$(7)$</td>
</tr>
<tr>
<td>TOP10</td>
<td>recA1 araD139 (MarA-1eu) 7697</td>
<td>Invitrogen</td>
</tr>
<tr>
<td>Rev1</td>
<td>sp-1 nlp12</td>
<td>Spontaneously from JE7951</td>
</tr>
<tr>
<td>Rev2</td>
<td>sp-1 nlp12</td>
<td>Spontaneously from JE7951</td>
</tr>
<tr>
<td>Rev3</td>
<td>sp-1 nlp13</td>
<td>Spontaneously from JE7951</td>
</tr>
<tr>
<td>Rev4</td>
<td>sp-1 nlp14</td>
<td>Spontaneously from JE7951</td>
</tr>
<tr>
<td>Rev5</td>
<td>sp-1 nlp15</td>
<td>Spontaneously from JE7951</td>
</tr>
<tr>
<td>Rev6</td>
<td>sp-1 nlp16</td>
<td>Spontaneously from JE7951</td>
</tr>
<tr>
<td>TAR1</td>
<td>str xyl argG2 sqj-203::Tn10</td>
<td>CAG12072 (P1) : JE7456</td>
</tr>
<tr>
<td>TAR2</td>
<td>argG sqj-203::Tn10</td>
<td>TAR1 (P1) : W3110</td>
</tr>
<tr>
<td>TAR3</td>
<td>sp-1 nlp2</td>
<td>TAR1 (P1) : JE7951</td>
</tr>
<tr>
<td>KM354</td>
<td>sp-1 nlp223 gam bet eco</td>
<td>$(7)$</td>
</tr>
<tr>
<td>MU2</td>
<td></td>
<td>Derived from KM354 by transformation with linearized nlp1::cm DNA</td>
</tr>
<tr>
<td>MU24</td>
<td>nlp1::cm</td>
<td>MU2 (P1) : W3110</td>
</tr>
<tr>
<td>MU25</td>
<td>nlp11</td>
<td>Rev1 (P1) : TAR2</td>
</tr>
<tr>
<td>MU26</td>
<td>nlp12</td>
<td>Rev2 (P1) : TAR2</td>
</tr>
<tr>
<td>MU27</td>
<td>nlp13</td>
<td>Rev3 (P1) : TAR2</td>
</tr>
<tr>
<td>MU124</td>
<td>sp-1 nlp1::cm</td>
<td>MU2 (P1) : JE7951</td>
</tr>
<tr>
<td>MU125</td>
<td>sp-1 nlp11</td>
<td>Rev1 (P1) : TAR3</td>
</tr>
<tr>
<td>MU126</td>
<td>sp-1 nlp12</td>
<td>Rev2 (P1) : TAR3</td>
</tr>
<tr>
<td>MU127</td>
<td>sp-1 nlp13</td>
<td>Rev3 (P1) : TAR3</td>
</tr>
<tr>
<td>PN1</td>
<td>prC $\Delta:$ neo nlp1::cm</td>
<td>MU24 (P1) : JE7934</td>
</tr>
<tr>
<td>HSB1</td>
<td>pB1::cm</td>
<td>Derived from KM354 by transformation with linearized ibpB::cm DNA</td>
</tr>
<tr>
<td>HSB2</td>
<td>pB1::cm</td>
<td>HS81 (P1) : W3110</td>
</tr>
<tr>
<td>HSB3</td>
<td>sp-1 ibpB::cm</td>
<td>HS81 (P1) : JE7951</td>
</tr>
</table>(underlined) 5'-TGGCGCGATGGAGGCCTTTTTGCG3-3' and 5'-GTCCACATGGCGGTGTCGGATTGTCGCG-3'. The amplified DNA was digested with NeOL, then inserted into the same site of pBAD Myc-His C. To construct pN16 encoding the hexa-histiadine-tagged Nlp1 with the amino-acid substitution G103D, the mutated nlp1 in the chromosomal DNA of Rev3 (spr-1 nlp13) was amplified by PCR using the primers described above, and the amplified DNA was amplified by PCR using the PCR using the PCR using the same site of pBAD Myc-His C. To construct pN17 encoding the truncated and hexa-histiadine-tagged Nlp1, the fragment containing the truncated nlp1 was amplified by PCR using the forward primer 5'-TGGCGGCATGGAGGCCTTTTTGCG-3' and the reverse primer 5'-GTGCATGGGGTCTGCCGC-3'. The amplified DNA was digested with NeOL and Nlp1 was identified to the same site of pBAD Myc-His C. To construct pN18 and pN19 were not found in the presence of the pN17 in pN17 in pN17 in pN17 in pN17. The amplified gene that was amplified as pN17 except that 5'-CATCCATGGCCTGGCGCCAGG-3' or 5'-CTTCCATGGGGCCAGGAGG-3', respectively, measured as the reverse primer in PCR. To construct pN110 encoding Nlp1, the nlp1 in the chromosomal DNA of W3110 was amplified by PCR using the forward primer containing the Neol site (underlined) 5'-TGGCGCATGGAGGCCTTTTTGCG-3'. The amplified DNA was amplified by the examining the BamH1 site (underlined) 5'-TGGATCGCGGTACAATGGAGG-3'. The amplified DNA was digested with NeOL and BamH1 and inserted into the same sites of pTrC99a. PN112, pN113, and pN114 were constructed in the same way as pN110 except that 5'-CCAGGATGCCTATGTCGTCCG-3', 5'-GTCCAGGATCCTATGGCGCAC- 3', or 5'-ATCTGGCATCTAGGCCAGGAG-3' are, as we used as the reverse primer containing the BamHI indicated (underlined) in PCR. The amplified DNA was digested with NeOL and BamHI and then inserted into the same sites of pTrC99a.
Expression and Purification of Hexa-Histidine-Tugged Nlp1 Protein-E, coli cells were transformed with the recombinant plasmids. This is the most conformationants were grown in the presence of the nimbated in L both components containing \(0.01 \%\) arabinase at \(42^{\circ} \mathrm{C}\) for \(4 \mathrm{~h}\) for expression of the fusion genes. Cells from \(400 \mathrm{ml}\) of medium were collected, washed once with ice-cold \(50 \mathrm{~mM}\) sodium phosphate buffer ( \(\mathrm{pH} 8.0\) ) containing \(300 \mathrm{mM} \mathrm{NaCl}\) (buffer A, suspended in \(4 \mathrm{ml}\) of the buffer A containing \(10 \mathrm{mM}\) imidazole and \(1 \%\) Triton X-100, and disrupted by sonication of the Nlp1 protein (see a mixture was centrifuged for \(15 \mathrm{~min}\) at \(17,700 \mathrm{~g}\) to the Nlp1 protein) and the Nlp1 1175 milliliters of the supernatant was mixed with \(1 \mathrm{ml}\) of nickelnitrolriacetic acid ( \(\mathrm{Ni}\)-TNA) in and incubated with rotation for \(2 \mathrm{~h}\) at \(4^{\circ} \mathrm{C}\). The mixture was loaded onto an open column. The column was washed with \(4 \mathrm{ml}\) each of buffer A containing \(20 \mathrm{mM}\) imidazole and then containing \(40 \mathrm{mM}\) imidazole. The bound proteins were eluted with \(1.2 \mathrm{ml}\) of buffer A containing \(100 \mathrm{mM}\) imidazole, and \(4 \mathrm{~AcaA} 4 \mathrm{G} \mathrm{G} 3^{\prime}\). The amplified DNA was digested with the mass Spectrometer (LQC DCE ACV P10 remain was另一 stromers. Thermo Terecht) coupled online with a capillary HPLC (Magic 2002, Michrom BioResources) was used to acquire MS/MS spectra. A \(0.2 \times 50 \mathrm{~mm}\) MAGIGMCS C18