TABLE 3. Mapping of transposon insertions sites that result in white phenotype in C. sakazakii ES5
<table border=1>
<tr>
<td rowspan="2">COG functional category ${}^{a}$
</td>
<td rowspan="2">COG functional class</td>
<td colspan="2">Annotation ${}^{b,c}$</td>
</tr>
<tr>
<td>Homologue</td>
<td>Gene product</td>
</tr>
<tr>
<td>
<table>
<tr>
<td>Mutation in pigment operon</td>
</tr>
<tr>
<td>
$\mathrm{H}$ : coenzyme transport and</td>
</tr>
<tr>
<td>
$\mathrm{H}$ : macrophages</td>
</tr>
<tr>
<td>GC: carbohydrate transport and</td>
</tr>
<tr>
<td>metabolism/signal</td>
</tr>
<tr>
<td>transduction mechanisms</td>
</tr>
<tr>
<td>RE: general function</td>
</tr>
<tr>
<td>
$\mathrm{H}$ : macrophages into acid</td>
</tr>
<tr>
<td>transport and metabolism</td>
</tr>
<tr>
<td>
$\mathrm{Q}$ : secondary metabolites</td>
</tr>
<tr>
<td>biosynthesis, transport and</td>
</tr>
<tr>
<td>catabolism</td>
</tr>
<tr>
<td>I: lipid transport and</td>
</tr>
<tr>
<td>molecular</td>
</tr>
<tr>
<td>Mutation outside pigment operon</td>
</tr>
<tr>
<td>C: energy production and</td>
</tr>
<tr>
<td>conversion</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>Geranylgeranyl pyrophosphate synthase</td>
</tr>
<tr>
<td>Glucosyl transferases, related to UDP-</td>
</tr>
<tr>
<td>glucosyltransferase</td>
</tr>
<tr>
<td>Acetyltransferase/choline</td>
</tr>
<tr>
<td>dehydrogenase and related</td>
</tr>
<tr>
<td>flavoproteins</td>
</tr>
<tr>
<td>Phytone dehydrogenase and related</td>
</tr>
<tr>
<td>proteins</td>
</tr>
<tr>
<td>Phytone/squalene synthase</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>crtE</td>
</tr>
<tr>
<td>crtX</td>
</tr>
<tr>
<td>crtY</td>
</tr>
<tr>
<td>crtY</td>
</tr>
<tr>
<td>crtI</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>GeranylGERanyl pyrophosphate</td>
</tr>
<tr>
<td>synthase</td>
</tr>
<tr>
<td>Zeaxanthin glucosyl transferase</td>
</tr>
</table>
</td>
</tr>
<tr>
<td>
<table>
<tr>
<td>F $\mathrm{F}_{1}$-type ATP synthase, subunit alpha</td>
</tr>
<tr>
<td>F $\mathrm{F}_{1}$-type ATP synthase, subunit beta</td>
</tr>
<tr>
<td>F $\mathrm{F}_{1}$-type ATP synthase, subunit</td>
</tr>
<tr>
<td>alpha</td>
</tr>
<tr>
<td>F $\mathrm{F}_{1}$-type ATP synthase, subunit</td>
</tr>
<tr>
<td>epsilon</td>
</tr>
<tr>
<td>F $\mathrm{F}_{1}$-type ATP synthase, subunit</td>
</tr>
<tr>
<td>epislon (mitochondrial delta subunit)</td>
</tr>
<tr>
<td>F $\mathrm{F}_{1}$-type ATP synthase, progenase</td>
</tr>
<tr>
<td>complex, dihydrolipoamide</td>
</tr>
<tr>
<td>acetyltransferase (E1) component,</td>
</tr>
<tr>
<td>dihydrogenase</td>
</tr>
<tr>
<td>Pyruvate/2-oxoglutarate dehydrogenase</td>
</tr>
<tr>
<td>complex, dihydrolipoamide</td>
</tr>
<tr>
<td>acetyltransferase (E2) component,</td>
</tr>
<tr>
<td>dihydrogenase</td>
</tr>
<tr>
<td>Pyruvate/2-oxoglutarate dehydrogenase</td>
</tr>
<tr>
<td>complex, dihydrolipoamide</td>
</tr>
<tr>
<td>acetyltransferase</td>
</tr>
<tr>
<td>acetyltransferase (E3) component,</td>
</tr>
<tr>
<td>and related enzyme</td>
</tr>
<tr>
<td>C-3-oxoglutarate dehydrogenases</td>
</tr>
<tr>
<td>Macridate dehydrogenase/fumarate</td>
</tr>
<tr>
<td>reductase, flavoprotein subunit</td>
</tr>
<tr>
<td>
$\mathrm{Na}^{+}/\mathrm{H}^{+}$antiporter</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>ESA_04012</td>
</tr>
<tr>
<td>ESA_04006</td>
</tr>
<tr>
<td>ESA_04007</td>
</tr>
<tr>
<td>ESA_04005</td>
</tr>
<tr>
<td>ESA_02622</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>F $\mathrm{F}_{1}$-F1 ATP synthase subunit alpha</td>
</tr>
<tr>
<td>
$\mathrm{F}_{1}$-F1 ATP synthase subunit beta</td>
</tr>
<tr>
<td>
$\mathrm{F}_{1}$-F1 ATP synthase subunit</td>
</tr>
<tr>
<td>alpha</td>
</tr>
<tr>
<td>F $\mathrm{F}_{1}$-F1 ATP synthase subunit</td>
</tr>
<tr>
<td>epsilon</td>
</tr>
<tr>
<td>SCa2-4-oxoglutarate</td>
</tr>
<tr>
<td>dehydrogenase E1 component</td>
</tr>
</table>
</td>
<td></td>
</tr>
<tr>
<td>
<table>
<tr>
<td>P: inorganic ion transport and</td>
</tr>
<tr>
<td>metabolism</td>
</tr>
<tr>
<td>T: signal transduction</td>
</tr>
<tr>
<td>mechanisms</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>cAMP-binding proteins, catabolite gene</td>
</tr>
<tr>
<td>activator, and regulatory subunit of</td>
</tr>
<tr>
<td>cAMP-dependent protein kinases</td>
</tr>
<tr>
<td>DnaK suppressor protein</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>ESA_03316</td>
</tr>
<tr>
<td>ESA_04376</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>acef $F$ dihydrolipoamide</td>
</tr>
<tr>
<td>acetyltransferase</td>
</tr>
</table>
</td>
</tr>
<tr>
<td>S: function unknown</td>
<td>Uncharacterized conserved protein</td>
<td>
<table>
<tr>
<td>ESA_03194</td>
</tr>
<tr>
<td>ESA_04343 (Ent6338_211)F</td>
</tr>
<tr>
<td>ESA_04343 (Ent6338_311)F</td>
</tr>
<tr>
<td>ESA_03563 (ETA_03450) ${}^{d}$
</td>
</tr>
<tr>
<td>ESA_00549 (AAG53883) ${}^{e}$
</td>
</tr>
</table>
</td>
<td>
<table>
<tr>
<td>DnaK transcriptional regulator</td>
</tr>
<tr>
<td>DksA</td>
</tr>
<tr>
<td>Acetallic protein (intracell</td>
</tr>
<tr>
<td>growth attenuator IgA.</td>
</tr>
<tr>
<td>Enterobacter sp. 638) ${}^{f}$
</td>
</tr>
<tr>
<td>Hypothetical protein (YhbC-like</td>
</tr>
<tr>
<td>protein, Envinia mutanensis</td>
</tr>
<tr>
<td>Hypothetical protein (sigma factor</td>
</tr>
<tr>
<td>RpoS, Escherichia coli) ${}^{g}$
</td>
</tr>
</table>
</td>
</tr>
</table>\footnotetext{
\({ }^{a}\) NCBI clusters of orthologous groups (COG) of proteins.
\({ }^{b}\) Cronobacter sakazakii ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).
\({ }^{c}\) NCBI assembly ATCC BAA-934 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000783.1).
\({ }^{d}\) Close annotated homolog.
}
(data not shown), consistently with the results for \(\mathrm{LB}\), all mutant strains showed significantly increased maximum rates of \(\operatorname{group}\left(\mu \mathrm{m} \alpha_{\mathrm{xx}}, 0.14 ; \mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{\text {crx }}, 0.22 ; \mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{c x} \mathrm{cr}, 0.24\); and \(\mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{\text {(crx) }} 0.19 ; P=0.000\).
\({ }^{f}\) Cold stress experiments were performed by growing \(\Delta \mathrm{cr} E\), \(\Delta \mathrm{crX}\), and \(\Delta \mathrm{crY}\) in \(\mathrm{LB}\) and \(\mathrm{Mg}\) medium at \(10^{\circ} \mathrm{C}\) (for results in \(\mathrm{Mg}\), see Fig. 1B). Under these conditions, no significant dif- ferences in maximum specific growth rates were detected for \(\Delta \mathrm{crY}, \Delta \mathrm{crH}, \Delta\) and \(\Delta \mathrm{crX}\) in \(\mathrm{b}\) and \(\mathrm{M} 9\) compared to those of with the wild type (in LB, \(\mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{c x}\) to \(\left.0.04, \mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{\mathrm{crX}}=0.03\right.\), \(\mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{(c x)}=0.03\), and \(\mu \mathrm{m} \alpha_{\mathrm{xx}}{ }^{(c x)}=0.04\); in \(\mathrm{Mg}, \mu \mathrm{m} \alpha_{\mathrm{xx}}{ }^{(c y)}=\) \(0.01, \mu \mathrm{m} \alpha_{\mathrm{xx}}{ }_{(c r X}=0.01, \mu \mathrm{m} \alpha_{\mathrm{xx} \mathrm{crY}}=0\), \(\Delta \mathrm{crX}\), and \(\Delta \mathrm{c} \mathrm{crY}\) in \(\mathrm{LB}\) and \(\mathrm {Mg}\) medium at \(10^{\circ} \mathrm{C}\) for results in \(\mathrm{D}\).
To evaluate growth under acidic conditions, wild-type and