# Y-Chromosome Variation in Southern African Khoe-San Populations Based on Whole-Genome Sequences  

Thijessen Naidoo12.34,t, Jingzi  $X_{\mathsf{U}}^{1,\dagger}$  Mario Vicente, Helena Malmstrm5, Himla Soodyall.7. , Mattias Jakobsson1.3.5, and Carina M. Schlebusch 1.35,\*  

Human Evolution, Department of Organismal Biology, Evolutionary Biology Centre, Uppsala University, Sweden   
2Department of Archaeology and Classical Studies, Stockholm University, Sweden   
3science for Life Laboratory, Uppsala, Sweden    
4Centre for Palaeogenetics, Stockholm, Sweden    
5PalaeesearchIstituteniversityf Jhanneurg,Auckland ar,outhfri   
Divisionf aneneticholfPathlogyFacultfHalthcinceiversityf theWitwaterrandaneurgouthi   
7National Health Laboratory Service, Johannesburg, South Africa   
8Academy of Science of South Africa  

\*Corresponding author:E-mail:carina.schlebusch@ebc.uu.se.  

Accepted:12May2020 These authorscontributed equally to thiswork  

Data deposition: The complete Y-chromosome sequences were deposited on the European Genome Phenome Archive (https://www.ebi.ac.uk/ ega/), accession number EGAs00001004459, and are available for researchuse under controlled access policies.  

# Abstract  

Although the human Y chromosome has effectively shown utlity in uncovering facets of human evolution and population histories, the ascertainment bias present in earlyY-chromosome variantdata sets limited the accuracy of diversity and TMRCA estimates obtained from them.The advent of next-generation sequencing,however,hasremoved thisbias and allwed for the discoveryof thousands of new variants for use in improving the Y-chromosome phylogeny and computing estimates that are more accurate. Here, we describe the high-coverage sequencing of the whole Y chromosome in a data set of 19 male Khoe-San individuals in comparisonwith existing whole-chromosome sequence data.Duetothe increasedresolution,wepotentiallyresolve thesourceof haplogroup B-P70 in the Khoe-San, and reconcile recently published haplogroup A-M51 data with the most recent version of the ISOGGY-chromosome phylogeny. Our results also improve the positioning of tentatively placed new branches of the ISOGG Ychromosome phylogeny.The distribution of major Y-chromosome haplogroups in theKhoe-San and other African groups coincide with the emerging picture of African demographic history;with E-M2 linked to the agriculturalist Bantu expansion, E-M35 linked to pastoralist easternAfricanmigrations,B-M112 linked to earliereast-south gene flow,A-M14 linked to shared ancestrywith central African rainforest hunter-gatherers, and A-M51 potentially unique to the Khoe-San.  

# Introduction  

The male-specific portion of the Y chromosome (MSY) haslongbeenregarded as aneffectivetool inthestudy ofhuman evolutionary history(Underhill and Kivisild 2007). It has proved useful mainly due to a lack of recombination along its length, making it the longest haplotypicblockin thehumangenome(Scozzari et al. 2012);and itspaternal mode of inheritance.The transmission of anintact haplotypefromfatherto son,changing only through mutation,preserves a simplerrecord of itshistory and allowsustostudythemalecontribution to the shaping of humanity.  