Table1.Datacollectionandrefinementstatistics   


<td><table  border="1"><thead><tr><td></td><td><b>Native</b></td><td><b>Xe derivative</b></td><td><b>Hg derivative</b></td><td><b>Pt derivative</b></td></tr></thead><tbody><tr><td>Resolution range (A)</td><td>20-1.9 (1.97-1.9)</td><td>30-2.0 (2.07-2.0)</td><td>50-2.80 (2.9-2.8)</td><td>30-3.1(3.24-3.1)</td></tr><tr><td>Unique reflections</td><td>42,743</td><td>37,784</td><td>12,595</td><td>10,230</td></tr><tr><td>Redundancy</td><td>3.0 (1.4)</td><td>9.2 (3.5)</td><td>3.4 (2.1)</td><td>2.1 (1.9)</td></tr><tr><td>Completeness (%)</td><td>93.2 (58.8)</td><td>98.2 (86.5)</td><td>90.1 (77.0)</td><td>97.3 (91.6)</td></tr><tr><td>Average I/o()</td><td>28.6 (2.5)</td><td>33.5 (2.2)</td><td>17.5 (3.8)</td><td>12.7 (1.5)</td></tr><tr><td>Rmerge (%)</td><td>4.4 (22.1)</td><td>7.0 (46.3)</td><td>6.0 (20.5)</td><td>8.1 (25.6)</td></tr><tr><td colspan="5">Refined model</td></tr><tr><td>Rfree (all data, 20-1.9 A) (%)</td><td>26.28</td><td></td><td></td><td></td></tr><tr><td>Reryst (all data, 20-1.9 A) (%)</td><td>20.31</td><td></td><td></td><td></td></tr><tr><td colspan="5">rmsd values</td></tr><tr><td>Bond lengths (A)</td><td>0.019</td><td></td><td></td><td></td></tr><tr><td>Bond angles (deg.)</td><td>1.691</td><td></td><td></td><td></td></tr></tbody></table></td>  

6 proteins, we incubated increasing amounts of each protein with a labelled DNA fragment carrying E2 site 4 from the HPV 16 genome and used DNase I to footprint the resulting complexes (Figure 4(a)). Both proteins protect a similar length of DNA from digestion by DNase I. Furthermore, the concentrations ofprotein required to obtain a clear footprint are roughly equivalent. Hydroxylradical  fotprinting   revealed   no  significant differences in the DBD-DNA complexes formed by the HPV 16 and HPV 6 DBDs (data not shown). Circular permutation assays using a set of DNA fragments in which the position of the E2 binding site is varied were used to investigate the effects of each DBD on DNA bending. On binding to DNA, both proteins induce DNA bending and there is no significant differencein the DNA bend angles:  $30(\pm1)^{\circ}$  for the HPV 16 DBD and  $27(\pm3)^{\circ}$  for the HPV 6 DBD (Figure  $4(b),$  .Although these data clearly indicate that the DNA bend angle in the HPV 6 and HPV 16 DBD-DNA complexes are very similar, the difficulties inherent in relating DNA bend angles determined using this approach to bend angles observed in protein-DNA co-crystals, make it difficult to say with any certainty whether these values differ significantly from the DNA bend angle of  $43-51^{\circ}$  seen in crystals of the HPV 18 and BPV1 E2 DBD-DNA complexes.7 Since the uncomplexed HPV 6 DBD superimposes on the HPV 18 and BPV1 DBDs, these data raise the possibility that the  $7\,\mathring{A}$  displacement observed in the structure of the HPV 16 DBD may not be replicatedin the HPV 16 E2 DBD-DNA complex. In summary, the data from gel retardation  assays  and  footprinting  experiments suggest that theHPV6and HPV16E2DBDsbind to a consensus E2 site with similar affinity, and that the protein-DNA complexes have similar architecture.  

# HPV6 andHPV16E2bindpreferentially to siteswith anA:T-rich spacer  

Unlike theBPVE2DBD, theDBDfrom theHPV 16 and HPV 18 E2 proteins bind preferentially to sites with an A:T-rich central spacer.712 Since the spacer region is not contacted directly by the DBD, this difference in binding specificity must be due to the overall conformation orflexibility afforded by these base-pairs to the surrounding DNA. To compare the binding of the HPV 16 and HPV6E2DBDstoE2siteswithdifferentcentral base-pairs, we made a series of E2 binding sites with either AATT, ACGT, or CCGG in the spacer region.Table2 showstheaffinityoftheHPV16 and HPV 6 E2 DBD for these sites  $(K_{\mathrm{eq(app)}})$  Both proteinsbind preferentially to an E2sitewith an A:T-rich spacer (Table 2, line 1). Changing the central spacer to either ACGT or CCGG,decreases binding by a factor of around tenfold in the case of theHPV 16 DBD andbymore than1000-fold in thecaseoftheHPV6DBD.Thevaluesforthe HPV 16E2 DBD differ somewhatfrom the 30-fold degree of preference reported previously for the binding of this protein to E2 sites with these spacer regions.1Â² One possibility is that this could be a consequence ofthedifferenttypesofoligonucleotides used in each study.Whilst we have used pairs of short complementary oligonucleotides, the previous study made use of single long oligonucleotides that anneal intramolecularly to form a hairpin of double-stranded DNA.12  

# HPV16andHPV6E2recogniseanextended bindingsite  

An alignment of the four E2 binding sites presentwithin theHPV 16 LCRproduces a consensus sequencethatdiffersfromthatderivedfromthe  