

TABLE 3. Mapping of transposon insertions sites that result in white phenotype in \( C \) . sakazakii ES5

<table><tr><td rowspan="2">COG functional category \( {}^{a} \)</td><td rowspan="2">COG functional class</td><td colspan="2">\( {\text{Annotation}}^{b, c} \)</td></tr><tr><td>Homologue</td><td>Gene product</td></tr><tr><td colspan="4">Mutation in pigment operon</td></tr><tr><td>\( \mathrm{H} \) : coenzyme transport and metabolism</td><td>Geranylgeranyl pyrophosphate synthase</td><td>crtE</td><td>Geranylgeranyl pyrophosphate synthase</td></tr><tr><td>GC: carbohydrate transport and metabolism/signal transduction mechanisms</td><td>Glucosyl transferases, related to UDP- glucosyltransferase</td><td>crtX</td><td>Zeaxanthin glucosyl transferase</td></tr><tr><td>R/E: general function prediction only/amino acid transport and metabolism</td><td>Acetyltransferase/choline dehydrogenase and related flavoproteins</td><td>crtY</td><td>Lycopene cyclase</td></tr><tr><td>Q: secondary metabolites biosynthesis, transport and catabolism</td><td>Phytoene dehydrogenase and related proteins</td><td>crtl</td><td>Phytoene dehydrogenase</td></tr><tr><td>I: lipid transport and metabolism</td><td>Phytoene/squalene synthase</td><td>crtB</td><td>Phytoene synthase</td></tr><tr><td colspan="4">Mutation outside pigment operon</td></tr><tr><td rowspan="9">C: energy production and conversion</td><td>\( {\mathrm{F}}_{0}{\mathrm{\;F}}_{1} \) -type ATP synthase, subunit alpha</td><td>ESA 04012</td><td>\( {\mathrm{F}}_{\mathrm{o}}{\mathrm{F}}_{1} \) ATP synthase subunit alpha</td></tr><tr><td>\( {\mathrm{F}}_{0}{\mathrm{\;F}}_{1} \) -type ATP synthase, subunit beta</td><td>ESA 04006</td><td>\( {\mathrm{F}}_{\mathrm{o}}{\mathrm{F}}_{1} \) ATP synthase subunit beta</td></tr><tr><td>\( {\mathrm{F}}_{\mathrm{o}}{\mathrm{F}}_{1} \) -type ATP synthase, subunit gamma</td><td>ESA 04007</td><td>\( {\mathrm{F}}_{\mathrm{o}}{\mathrm{F}}_{1} \) ATP synthase subunit gamma</td></tr><tr><td>\( {\mathrm{F}}_{0}{\mathrm{\;F}}_{1} \) -type ATP synthase, subunit epsilon (mitochondrial delta subunit)</td><td>ESA_04005</td><td>\( {\mathrm{F}}_{0}{\mathrm{\;F}}_{1} \) ATP synthase subunit epsilon</td></tr><tr><td>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E1) component, and related enzymes</td><td>ESA 02622</td><td>sucA 2-oxoglutarate dehydrogenase E1 component</td></tr><tr><td>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E2) component, and related enzymes</td><td>ESA 02621</td><td>Dihydrolipoamide acetyltransferase</td></tr><tr><td>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E3) component, and related enzymes</td><td>ESA 03222</td><td>\( {aceF} \) dihydrolipoamide acetyltransferase</td></tr><tr><td>Malate/lactate dehydrogenases</td><td>ESA 03622</td><td>Malate dehydrogenase</td></tr><tr><td>Succinate dehydrogenase/fumarate reductase, flavoprotein subunit</td><td>ESA 02624</td><td>Succinate dehydrogenase flavoprotein subunit</td></tr><tr><td>P: inorganic ion transport and metabolism</td><td>\( {\mathrm{{Na}}}^{ + }/{\mathrm{H}}^{ + } \) antiporter</td><td>ESA 03316</td><td>pH-dependent sodium/proton antiporter</td></tr><tr><td rowspan="2">T: signal transduction mechanisms</td><td>cAMP-binding proteins, catabolite gene activator, and regulatory subunit of cAMP-dependent protein kinases</td><td>ESA_04376</td><td>cAMP regulatory protein</td></tr><tr><td>DnaK suppressor protein</td><td>ESA 03194</td><td>DnaK transcriptional regulator DksA</td></tr><tr><td>S: function unknown</td><td>Uncharacterized conserved protein</td><td>ESA 04343 (Ent638 3811)</td><td>Hypothetical protein (intracellular growth attenuator IgA. Enterobacter sp. 638) \( {}^{d} \)</td></tr><tr><td/><td/><td>ESA 03563 (ETA 03450)</td><td>Hypothetical protein (YhbC-like protein, Erwinia tasmaniensis Et1/99)</td></tr><tr><td/><td/><td>ESA 00549 (AAG53883)</td><td>Hypothetical protein (sigma factor RpoS, Escherichia coli)</td></tr></table>

\( {}^{a} \) NCBI clusters of orthologous groups (COG) of proteins.

\( {}^{b} \) Cronobacter sakazakii ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).

\( {}^{c} \) NCBI assembly ATCC BAA-894 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000783.1).

\( {}^{d} \) Closest annotated homolog.

(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth \( \left( {\mu \mathop{\max }\limits_{\text{wt }},{0.14};\mu \mathop{\max }\limits_{{\Delta \operatorname{crt}X}},{0.22};\mu \mathop{\max }\limits_{{\Delta \operatorname{crt}E}},{0.24}}\right. \) ; and \( \left. {\mu \mathop{\max }\limits_{{\Delta \operatorname{cr}{tY}}},{0.19};P = {0.000}}\right) \) .

Cold stress experiments were performed by growing \( \Delta \operatorname{crt}E \) , \( {\Delta crtX} \) , and \( {\Delta crtY} \) in \( \mathrm{{LB}} \) and \( \mathrm{M}9 \) medium at \( {10}^{ \circ  }\mathrm{C} \) (for results in M9, see Fig. 1B). Under these conditions, no significant differences in maximum specific growth rates were detected for \( {\Delta crtY},{\Delta crtE} \) , and \( {\Delta crtX} \) in both LB and M9 compared to those of the wild type (in \( \mathrm{{LB}},\mu \mathop{\max }\limits_{\mathrm{{wt}}} = {0.04},\mu \mathop{\max }\limits_{{\Delta \operatorname{crt}X}} = {0.03} \) , \( \mu \mathop{\max }\limits_{{\Delta \text{ crtE }}} = {0.03} \) , and \( \mu \mathop{\max }\limits_{{\Delta \text{ crtY }}} = {0.04} \) ; in M9, \( \mu \mathop{\max }\limits_{\text{wt }} = \) \( {0.01},\mu \mathop{\max }\limits_{{\Delta crtX}} = {0.01},\mu \mathop{\max }\limits_{{\Delta crtE}} = {0.01} \) , and \( \mu \mathop{\max }\limits_{{\Delta crtY}} = \) \( {0.01}) \) .

To evaluate growth under acidic conditions, wild-type and