# JOHLER ET AL

# APPL. ENVIRON. MICROBIOL.

# TABLE 3 Mapping of transposon insertions sites that result in white phenotype in C sakazakii ESS

|COG functional category|COG functional class|Homologue|Annotation|Gene product|
|---|---|---|---|---|
|Mutation in pigment operon|H: coenzyme transport and metabolism|CrtE|Geranylgeranyl pyrophosphate synthase|Geranylgeranyl pyrophosphate synthase|
|GC: carbohydrate transport and metabolism/signal transduction mechanisms|crtX|Glucosyl transferases, related to UDP-glucosyltransferase|Zeaxanthin glucosyl transferase| |
|RIE: general function prediction only/amino acid transport and metabolism|crty|Acetyltransferase/choline dehydrogenase and related flavoproteins|Lycopene cyclase| |
|Q: secondary metabolites biosynthesis, transport and catabolism|crt[|Phytoene dehydrogenase and related proteins|Phytoene dehydrogenase| |
| |CrtB|Phytoene/squalene synthase|Phytoene synthase| |
|Mutation outside pigment operon|energy production and conversion|ESA 04012|FF-type ATP synthase, subunit alpha|FSF, ATP synthase subunit alpha|
| |ESA 04006|FoFu-type ATP synthase, subunit beta|FOF, ATP synthase subunit beta| |
| |ESA 04007|FoF-type ATP synthase, subunit gamma|FF, ATP synthase subunit gamma| |
| |ESA 04005|FoF-type ATP synthase, subunit epsilon (mitochondrial delta subunit)|epsilon| |
| |ESA 02622|Pyruvate/2-oxoglutarate dehydrogenase complex; dihydrolipoamide acetyltransferase (E1) component; and related enzymes| | |
| |ESA 02621|Pyruvate/2-oxoglutarate dehydrogenase complex; dihydrolipoamide acetyltransferase (E2) component; and related enzymes|ESA 03222| |
| |ESA 03622|Pyruvate/2-oxoglutarate dehydrogenase complex; dihydrolipoamide acetyltransferase (E3) component; and related enzymes| | |
| |ESA 02624|Malate/lactate dehydrogenases|Malate dehydrogenase| |
|P: inorganic ion transport and metabolism|ESA 03316|Na+/H antiporter|pH-dependent sodium proton antiporter| |
|T: signal transduction mechanisms|ESA 04376|cAMP-binding proteins|catabolite gene activator and regulatory subunit of cAMP-dependent protein kinases| |
| |ESA 03194|DnaK transcriptional regulator|DnaK suppressor protein| |
|function unknown|ESA_04343 (Ent638_3811)|Uncharacterized conserved protein|Hypothetical protein (intracellular growth attenuator IgA)| |
| |ESA_03563 (ETA_03450)|Hypothetical protein (YhbC-like protein. Erwvinia tasmaniensis)| | |
| |ESA_00549 (AAG53883)|Hypothetical protein (sigma factor RpoS Escherichia coli)| | |

orthologous groups (COG) of proteins

NCBI clusters of Cronobacter sakazakii ESS BAC 9EIO for mutations within the pigment operon (accession no. AM384990.1)

NCBI assembly ATCC BAA-894 C sakazakii complete genome for mutations outside pigment operon (accession no. CPO00783.1).

Closest annotated homolog: (data not shown), consistently with the results for LB all mutant strains showed significantly increased maximum rates of growth (umax,t; 0.14; pmaxAcrtxs 0.22; pmaxAcnEs0.24; and of the wild type (in LB, umaxwt 0.04, pmaxAcnE 0.03 and pmax_crtY 0.04; in M9, umaxwt 0.01. umaxscrX 0.01, pmaxAcnE 0.01, and umaxscnY 0.01).

Cold stress experiments were performed by growing AcrE; AcrtX, and AcrtY in LB and M9 medium at 10Â°C (for results in M9, see Fig: IB). Under these conditions, no significant differences in maximum specific growth rates were detected for AcrtY, AcrtE, and AcrtX in both LB and M9 compared to those of the wild type.