TABLE 3. Mapping of transposon insertions sites that result in white phenotype in C. sakazakii ES5

| COG functional category* | COG functional class | Annotation†‡ | Homologue | Gene product |
|--------------------------|----------------------|--------------|-----------|-------------|
| Mutation in pigment operon | Geranylgeranyl pyrophosphate synthase | crtE | Geranylgeranyl pyrophosphate synthase |
| H: coenzyme transport and metabolism | Glucosyl transferases, related to UDP-glucosyltransferase | crtX | Zeaxanthin glucosyl transferase |
| GC: carbohydrate transport and metabolism/signal transduction mechanisms | Acetyltransferase/choline dehydrogenase and related flavoproteins | cty | Lycopene cyclase |
| R/E: general function prediction only/amino acid transport and metabolism | Phytoene dehydrogenase and related proteins | cti | Phytoene dehydrogenase |
| Q: secondary metabolites biosynthesis, transport and catabolism | Phytoene/squalene synthase | ctB | Phytoene synthase |
| I: lipid transport and metabolism |                     |             |           |             |
| Mutation outside pigment operon |                     |             |           |             |
| C: energy production and conversion | F2, F2-type ATP synthase, subunit alpha | ESA_04012 | F2, F2 ATP synthase subunit alpha |
| | F2, F2-type ATP synthase, subunit beta | ESA_04006 | F2, F2 ATP synthase subunit beta |
| | F2, F2-type ATP synthase, subunit gamma | ESA_04007 | F2, F2 ATP synthase subunit gamma |
| | F2, F2-type ATP synthase, subunit epsilon (mitochondrial delta subunit) | ESA_04005 | F2, F2 ATP synthase subunit epsilon |
| | Pyrurate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E1) component, and related enzymes | ESA_02622 | acce/2-oxoglutarate dehydrogenase E1 component |
| | Pyrurate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E2) component, and related enzymes | ESA_02621 | Dihydrolipoamide acetyltransferase |
| | Pyrurate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E3) component, and related enzymes | ESA_03222 | acef Dihydrolipoamide acetyltransferase |
| | Malate/lactate dehydrogenases | ESA_03622 | Malate dehydrogenase |
| | Succinate dehydrogenases/fumarate reductase, flavoprotein subunit | ESA_02624 | Succinate dehydrogenase/fluoroprotein subunit |
| | Na+ /H+ antiporter | ESA_03316 | pH-dependent sodium/proton antiporter |
| P: inorganic ion transport and metabolism | cAMP-binding proteins, catabolite gene activator, and regulatory subunit of cAMP-dependent protein kinases | ESA_04376 | cAMP regulatory protein |
| T: signal transduction mechanisms | Supernumerary 32-kDa homolog of dentrocidin effector 32 | ESA_03914 | DnaK transcriptional regulator DksA |
| S: function unknown | Uncharacterized conserved protein | ESA_04343 (Ent638_3811) | Hypothetical protein (intracellular growth attenuator IgA, Enterobacter sp. 638) |
| | ESA_03563 (ETA_03450) | Hypothetical protein (YhhC-like protein, Erwinia tasmaniensis E1/99) |
| | ESA_00549 (AAG53883) | Hypothetical protein (sigma factor RpoS, Escherichia coli) |

* NCBI clusters of orthologous groups (COG) of proteins.
† Cronobacter sakazakii ESS BAC 9E10 for mutations within the pigment operon (accession no. AM386900.1).
‡ NCBI Assembly ATCC BAA-894 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000073.1).
§ Closest annotated homolog.

(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth ($\mu_{max}$, 0.14; $\mu_{max}$, 0.22; $\mu_{max}$, 0.24; and $\mu_{max}$, 0.19; $P = 0.000$).

Cold stress experiments were performed by growing $\Delta crE$, $\Delta crX$, and $\Delta crY$ in LB and M9 medium at 10°C (for results in M9, see Fig. 1B). Under these conditions, no significant differences in maximum specific growth rates were detected for $\Delta crY$, $\Delta crE$, and $\Delta crX$ in both LB and M9 compared to those of the wild type (in LB, $\mu_{max}$ = 0.04, $\mu_{max}$, 0.03, $\mu_{max}$, 0.03, and $\mu_{max}$, 0.04; in M9, $\mu_{max}$ = 0.01, $\mu_{max}$, 0.01, $\mu_{max}$, 0.01, and $\mu_{max}$, 0.01).

To evaluate growth under acidic conditions, wild-type and