```markdown
1056 JOHLER ET AL.
APPL. ENVIRON. MICROBIOL.
TABLE 3. Mapping of transposon insertions sites that result in white phenotype in C. sakazakii ES5
<table>
<thead>
<tr>
<th>COG functional category"</th>
<th>COG functional class</th>
<th>Homologue</th>
<th>Annotation<sup>b.c</sup></th>
<th>Gene product</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mutation in pigment operon</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>H: coenzyme transport and
metabolism</td>
<td>Geranylgeranyl pyrophosphate synthase</td>
<td><i>crtE</i></td>
<td></td>
<td>Geranylgeranyl pyrophosphate synthase</td>
</tr>
<tr>
<td>GC: carbohydrate transport and
metabolism/signal
transduction mechanisms</td>
<td>Glucosyl transferases, related to UDP-
glucosyltransferase</td>
<td><i>crtX</i></td>
<td></td>
<td>Zeaxanthin glucosyl transferase</td>
</tr>
<tr>
<td>R/E: general function
prediction only/amino acid
transport and metabolism</td>
<td>Acetyltransferase/choline
dehydrogenase and related
flavoproteins</td>
<td><i>crtY</i></td>
<td></td>
<td>Lycopene cyclase</td>
</tr>
<tr>
<td>Q: secondary metabolites
biosynthesis, transport and
catabolism</td>
<td>Phytoene dehydrogenase and related
proteins</td>
<td><i>crtI</i></td>
<td></td>
<td>Phytoene dehydrogenase</td>
</tr>
<tr>
<td>I: lipid transport and
metabolism</td>
<td>Phytoene/squalene synthase</td>
<td><i>crtB</i></td>
<td></td>
<td>Phytoene synthase</td>
</tr>
<tr>
<td>Mutation outside pigment operon</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>C: energy production and
conversion</td>
<td>FoF₁-type ATP synthase, subunit alpha</td>
<td>ESA_04012</td>
<td></td>
<td>FoF₁ ATP synthase subunit alpha</td>
</tr>
<tr>
<td></td>
<td>FoF₁-type ATP synthase, subunit beta</td>
<td>ESA_04006</td>
<td></td>
<td>FoF₁ ATP synthase subunit beta</td>
</tr>
<tr>
<td></td>
<td>FoF₁-type ATP synthase, subunit
gamma</td>
<td>ESA_04007</td>
<td></td>
<td>FoF₁ ATP synthase subunit
gamma</td>
</tr>
<tr>
<td></td>
<td>FoF₁-type ATP synthase, subunit
epsilon (mitochondrial delta subunit)</td>
<td>ESA_04005</td>
<td></td>
<td>FoF₁ ATP synthase subunit
epsilon</td>
</tr>
<tr>
<td></td>
<td>Pyruvate/2-oxoglutarate dehydrogenase
complex, dihydrolipoamide
acetyltransferase (E1) component,
and related enzymes</td>
<td>ESA_02622</td>
<td></td>
<td><i>sucA</i> 2-oxoglutarate
dehydrogenase E1 component</td>
</tr>
<tr>
<td></td>
<td>Pyruvate/2-oxoglutarate dehydrogenase
complex, dihydrolipoamide
acetyltransferase (E2) component,
and related enzymes</td>
<td>ESA_02621</td>
<td></td>
<td>Dihydrolipoamide
acetyltransferase</td>
</tr>
<tr>
<td></td>
<td>Pyruvate/2-oxoglutarate dehydrogenase
complex, dihydrolipoamide
acetyltransferase (E3) component,
and related enzymes</td>
<td>ESA_03222</td>
<td></td>
<td><i>aceF</i> dihydrolipoamide
acetyltransferase</td>
</tr>
<tr>
<td></td>
<td>Malate/lactate dehydrogenases</td>
<td>ESA_03622</td>
<td></td>
<td>Malate dehydrogenase</td>
</tr>
<tr>
<td></td>
<td>Succinate dehydrogenase/fumarate
reductase, flavoprotein subunit</td>
<td>ESA_02624</td>
<td></td>
<td>Succinate dehydrogenase
flavoprotein subunit</td>
</tr>
<tr>
<td>P: inorganic ion transport and
metabolism</td>
<td>Na+/H+ antiporter</td>
<td>ESA_03316</td>
<td></td>
<td>pH-dependent sodium/proton
antiporter</td>
</tr>
<tr>
<td>T: signal transduction
mechanisms</td>
<td>CAMP-binding proteins, catabolite gene
activator, and regulatory subunit of
CAMP-dependent protein kinases</td>
<td>ESA_04376</td>
<td></td>
<td>cAMP regulatory protein</td>
</tr>
<tr>
<td></td>
<td>Dnak suppressor protein</td>
<td>ESA_03194</td>
<td></td>
<td>DnaK transcriptional regulator
DksA</td>
</tr>
<tr>
<td>S: function unknown</td>
<td>Uncharacterized conserved protein</td>
<td>ESA_04343 (Ent638_3811)<sup>d</sup></td>
<td></td>
<td>Hypothetical protein (intracellular
growth attenuator IgA,
<i>Enterobacter</i> sp. 638)<sup>d</sup></td>
</tr>
<tr>
<td></td>
<td></td>
<td>ESA_03563 (ETA_03450)<sup>d</sup></td>
<td></td>
<td>Hypothetical protein (YhbC-like
protein, <i>Erwinia tasmaniensis</i>
Et1/99)<sup>d</sup></td>
</tr>
<tr>
<td></td>
<td></td>
<td>ESA_00549 (AAG53883)<sup>d</sup></td>
<td></td>
<td>Hypothetical protein (sigma factor
RpoS, <i>Escherichia coli</i>)<sup>d</sup></td>
</tr>
</tbody>
</table>
" NCBI clusters of orthologous groups (COG) of proteins.
b Cronobacter sakazakii ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).
c NCBI assembly ATCC BAA-894 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000783.1).
d Closest annotated homolog.

(data not shown), consistently with the results for LB, all mu-
tant strains showed significantly increased maximum rates of
growth (\(\mu\)maxwt, 0.14; \(\mu\)max \(\Delta crtX\), 0.22; \(\mu\)max \(\Delta crtE\), 0.24; and
\(\mu\)max \(\Delta crtY\), 0.19; \(P\) = 0.000).
Cold stress experiments were performed by growing \(\Delta crtE\),
\(\Delta crtX\), and \(\Delta crtY\) in LB and M9 medium at 10°C (for results in
M9, see Fig. 1B). Under these conditions, no significant dif-
ferences in maximum specific growth rates were detected for
\(\Delta crtY\), \(\Delta crtE\), and \(\Delta crtX\) in both LB and M9 compared to those
of the wild type (in LB, \(\mu\)maxwt = 0.04, \(\mu\)max \(\Delta crtX\) = 0.03,
\(\mu\)max \(\Delta crtE\) = 0.03, and \(\mu\)max \(\Delta crtY\) = 0.04; in M9, \(\mu\)maxwt =
0.01, \(\mu\)max \(\Delta crtX\) = 0.01, \(\mu\)max \(\Delta crtE\) = 0.01, and \(\mu\)max \(\Delta crtY\)
= 0.01).
To evaluate growth under acidic conditions, wild-type and
```