```markdown
| COG functional category* | COG functional class | Homologue | Annotation* | Gene product |
| --- | --- | --- | --- | --- |
| Mutation in pigment operon | | | | |
| H: coenzyme transport and metabolism | Geranylgeranyl pyrophosphate synthase | crtE | Geranylgeranyl pyrophosphate synthase |
| GC: carbohydrate transport and metabolism | Glucosyl transferases, related to UDP- | crtX | Zeaxanthin glucosyl transferase |
| I: lipid transport and metabolism | Phytoene/squalene synthase | crtB | Phytoene synthase |
| Mutation outside pigment operon | | | | |
| C: energy production and conversion | F,F,F-type ATP synthase, subunit alpha | ESA_04012 | F,F,F-type ATP synthase subunit alpha |
| | F,F,F-type ATP synthase, subunit beta | ESA_04006 | F,F,F-type ATP synthase subunit beta |
| | F,F,F-type ATP synthase, subunit gamma | ESA_04007 | F,F,F-type ATP synthase subunit gamma |
| | F,F,F-type ATP synthase, subunit epsilon | ESA_04005 | F,F,F-type ATP synthase subunit epsilon |
| | Pyruvate/2-oxoglutarate dehydrogenase complex, E1 component, subunit | ESA_02622 | succA2-oxoglutarate dehydrogenase E1 component |
| | Acetyltransferase (E1) component, and related enzymes | ESA_02621 | Dihydrolipoamide acetyltransferase |
| | Pyruvate/2-oxoglutarate dehydrogenase complex,dihydrodipamide acetyltransferase, and related enzymes | ESA_03222 | aceF dihydrolipoamide acetyltransferase |
| | Malate/lactate dehydrogenase | ESA_03622 | Malate dehydrogenase |
| | Succinate dehydrogenase/fumarate reductase flavoprotein subunit | ESA_06224 | Succinate dehydrogenase |
| | Na+/H+ antiporter protein subunit | ESA_03136 | Na+/H+ antiporter protein subunit |
| P: inorganic ion transport and metabolism | cAMP-binding proteins, catabolite activator, and regulatory subunit of cAMP-dependent protein kinase | ESA_04376 | cAMP regulatory protein |
| | DnaK suppressor protein | ESA_03194 | DnaK transcriptional regulator |
| S: function unknown | Uncharacterized conserved protein | ESA_04343 (Ent6381_111)* | Hypothetical protein (intracellular growth attenuator IgA) |
| | | ESA_03563 (ETA_04350)* | Hypothetical protein sp. 6387 |
| | | ESA_00549 (AAG53883)* | Hypothetical protein (Erwinia carotovora-like protein) |
| | | ESA_02699* | Hypothetical protein (sigma factor RpoS, Escherichia coli) |

*NCBI clusters of orthologous groups (COG) of proteins.
*Cronobacter sakazakii BAA-894 complete gene for mutations within the pigment operon (accession no. AM3849901).
*NCBI assembly ATCC BAA-894 C. sakazakii complete gene for mutations outside pigment operon (accession no. CP000831).
*Closest annotated homolog.

(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth (μmax) in both LB and M9 compared to those of the wild type (in LB, μmax = 0.14, μmax_crtX = 0.22, μmax_crtY = 0.24; and μmax_crtX = 0.19, P = 0.000; in M9, μmax = 0.04, μmax_crtX = 0.03, μmax_crtY = 0.04; μmax_crtX = 0.01, P = 0.000; μmax_crtY = 0.01, P = 0.000). Cold stress experiments were performed by growing ΔcrtE, ΔcrtX, and ΔcrtY in LB and M9 medium at 10°C (for results in M9, see Fig. 1B). Under these conditions, no significant differences in maximum specific growth rates were detected for ΔcrtE, ΔcrtX, and ΔcrtY in both LB and M9 compared to those of the wild type (in LB, μmax_wt = 0.04, μmax_crtX = 0.03, μmax_ crtY = 0.04; in M9, μmax_wt = 0.04, μmax_crtX = 0.
01, μmax_crtY = 0.01, P = 0.000; μmax_crtX = 0.01, P = 0.000; and μmax_crtY = 0.01, P = 0.000).

To evaluate growth under acidic conditions, wild-type and ΔcrtE, ΔcrtX, and ΔcrtY mutant strains were grown in LB and M9 medium at pH 5.5.
```