```markdown
# Recognition of DNA Conformation

## Table 1. Data collection and refinement statistics

| Native | Xe derivative | Hg derivative | Pt derivative |
|---------|---------------|---------------|---------------|
| Resolution range (Å) | 20–19 (19.7–19.9) | 30–20 (20.7–20.0) | 50–28 (29.8–28.8) | 30–31 (32.4–31.1) |
| Unique reflections | 42,743 | 37,794 | 12,595 | 10,230 |
| Redundancy | 3.0 (1.4) | 1.9 (0.9) | 1.4 (2.1) | 2.1 (1.9) |
| Completeness (%) | 93.2 (68.8) | 92.2 (35.5) | 90.1 (77.0) | 97.3 (91.6) |
| Average I/σ(I) | 4.8 (2.2) | 9.2 (2.5) | 17.5 (3.8) | 12.7 (1.5) |
| Rmerge (%) | 4.4 (22.1) | 7.0 (46.3) | 6.0 (20.5) | 8.1 (25.6) |
| Refined model | | | |
| R (all data, 20–19 Å) (%) | 26.28 | | | |
| Rfree (all data, 20–19 Å) (%) | 20.31 | | | |
| rmsd values | | | | |
| Bond lengths (Å) | 0.019 | | | |
| Bond angles (deg.) | 1.691 | | | |

6 proteins, we incubated increasing amounts of each protein with a labelled DNA fragment carrying E2 site I from the HPV 16 genome and used DNA footprinting to map the binding site of the E2 protein. The footprint from the HPV 16 E2 protein is shown in Figure 4(a). Both proteins protect a similar length of DNA from digestion by DNase I. Furthermore, the concentrations of protein required to clear a footprint are roughly equivalent. Hydroxyl radical footprinting revealed no significant differences in the DBD DNA footprint formed by the HPV 16 and HPV 6 DBDs (data not shown). Circular permutation assays using a set of DNA fragments in which the position of the E2 binding site is varied were used to investigate the binding of E2 to DNA. Both proteins induce DNA bending and there is no significant difference in the HPV 16 and HPV 6 DBD DNA bend angles (30 ± 3°). The HPV 6 DBD footprint is shown in Figure 4(b). Although these data clearly indicate that the DNA binding properties of the HPV 6 and HPV 16 DBD-DNA complexes are very similar, the difficulties inherent in relating bend angles observed in protein-DNA co-crystals to bend makes angles observed in protein-DNA co-crystals, whether it is valid to say significantly different from the DNA bend angle of 43–51° seen in crystals of the HPV 18 and HPV 1 DBD DBD complexes.7 One possibility is that this could be a consequence of the different types of oligonucleotide used in each study. Whilst we have used pairs of short complementary oligonucleotides, the previous study made use of single long oligonucleotides that anneal intramolecularly to form a hairpin of double-stranded DNA.12

HPV 16 and HPV 6 E2 recognise an extended binding site

An alignment of the four E2 binding sites present within the HPV 16 LCR produces a consensus sequence that differs from that derived from the HPV 16 and HPV 6 E2 DBD-DNA complexes have similar architecture.

(d) A comparison of the β2/β3 loop in the HPV 6 E2 DBD (blue) and BPV1 E2 DBD (yellow). BPV1 E2 Arg370 is indicated in pink. HPV 6 E2 Lys327, Lys323, Asp311 and His366 are indicated in light blue. His366 is present in two conformations in the crystal structure.
```