```markdown
# Table 1. Bacterial strains

<table>
    <tr>
        <th>Strain</th>
        <th>Relevant characteristics</th>
        <th>Source or reference</th>
    </tr>
    <tr>
        <td>W3110</td>
        <td>wild type</td>
        <td>Laboratory stock</td>
    </tr>
    <tr>
        <td>JE7951</td>
        <td>sp1?</td>
        <td>(7)</td>
    </tr>
    <tr>
        <td>JE7934</td>
        <td>prc1::cnc</td>
        <td>(7)</td>
    </tr>
    <tr>
        <td>JE7456</td>
        <td>stf yqxG</td>
        <td>Laboratory stock</td>
    </tr>
    <tr>
        <td>CAG12072</td>
        <td>zyg-203::Tn10</td>
        <td>(26)</td>
    </tr>
    <tr>
        <td>TOP10</td>
        <td>recA araD139 (MaraA-leu) 7697</td>
        <td>Invitrogen</td>
    </tr>
    <tr>
        <td>Rev1</td>
        <td>spr1 nlpI1</td>
        <td>Spontaneously from JE7951</td>
    </tr>
    <tr>
        <td>Rev2</td>
        <td>spr1 nlpI2</td>
        <td>Spontaneously from JE7951</td>
    </tr>
    <tr>
        <td>Rev3</td>
        <td>spr1 nlpI3</td>
        <td>Spontaneously from JE7951</td>
    </tr>
    <tr>
        <td>Rev4</td>
        <td>spr1 nlpI4</td>
        <td>Spontaneously from JE7951</td>
    </tr>
    <tr>
        <td>Rev5</td>
        <td>spr1 nlpI5</td>
        <td>Spontaneously from JE7951</td>
    </tr>
    <tr>
        <td>Rev6</td>
        <td>spr1 nlpI6</td>
        <td>Spontaneously from JE7951</td>
    </tr>
    <tr>
        <td>TAR1</td>
        <td>stf argG zgi-203::Tn10</td>
        <td>CAG12072 (P1) × JE7456</td>
    </tr>
    <tr>
        <td>TAR2</td>
        <td>argG zgi-203::Tn10</td>
        <td>TAR1 (P1) × W3110</td>
    </tr>
    <tr>
        <td>TAR3</td>
        <td>spr1 argG zgi-203::Tn10</td>
        <td>TAR1 (P1) × JE7951</td>
    </tr>
    <tr>
        <td>KM354</td>
        <td>rpsL?:prpT222 gam bet exo</td>
        <td>(14)</td>
    </tr>
    <tr>
        <td>MU2</td>
        <td>nlpI::xcm</td>
        <td>Derived from KM354 by transformation with linearized nlpI::xcm DNA</td>
    </tr>
    <tr>
        <td>MU21</td>
        <td>nlpI</td>
        <td>MU2 (P1) × W3110</td>
    </tr>
    <tr>
        <td>MU25</td>
        <td>nlpI1</td>
        <td>Rev1 (P1) × TAR2</td>
    </tr>
    <tr>
        <td>MU26</td>
        <td>nlpI2</td>
        <td>Rev2 (P1) × TAR2</td>
    </tr>
    <tr>
        <td>MU27</td>
        <td>nlpI3</td>
        <td>Rev3 (P1) × TAR2</td>
    </tr>
    <tr>
        <td>MU121</td>
        <td>spr1 nlpI</td>
        <td>MU3 (P1) × JE7951</td>
    </tr>
    <tr>
        <td>MU123</td>
        <td>spr1 nlpI1</td>
        <td>Rev1 (P1) × TAR3</td>
    </tr>
    <tr>
        <td>MU126</td>
        <td>spr1 nlpI2</td>
        <td>Rev2 (P1) × TAR3</td>
    </tr>
    <tr>
        <td>MU127</td>
        <td>spr1 nlpI3</td>
        <td>Rev3 (P1) × TAR3</td>
    </tr>
    <tr>
        <td>PN1</td>
        <td>prc::neo nlpI::cm</td>
        <td>MU24 (P1) × JE7934</td>
    </tr>
    <tr>
        <td>HSB1</td>
        <td>ibpB::cm</td>
        <td>Derived from KM354 by transformation with linearized ibpB::cm DNA</td>
    </tr>
    <tr>
        <td>HSB2</td>
        <td>ibpB::cm</td>
        <td>HSB1 (P1) × W3110</td>
    </tr>
    <tr>
        <td>HSB3</td>
        <td>spr1-ibpB::cm</td>
        <td>HSB1 (P1) × JE7951</td>
    </tr>
</table>

(underline), 5'-TGGCCACTGAGACCTTTTTTTTTGCG-3' and 5'-GTCACCAGCTGGTCCTAGACCAGCTGC-3'. The amplified DNA was digested with NcoI, then inserted into the same site of pBAD Myc-His C. To construct pNI6 encoding the hexahistidine-tagged NlpI with the amino-acid substitution G103D, the mutated nlpI in the chromosomal DNA of Rev6 (spr1 nlpI6) was amplified by PCR using the primers described above, and the amplified DNA was digested with NcoI and inserted into the same site of pBAD Myc-His C. To construct pNI7 encoding the truncated and hexahistidine-tagged NlpI, the fragment containing the truncated nlpI was amplified by PCR using the forward primer 5'-TGGCCACTGAGACCTTTTTTTTTTGCG-3' and the reverse primer 5'-GTCACATGGGTGCCCTGCGGC-3'. The amplified DNA was digested with NcoI, then inserted into the same site of pBAD Myc-His C. pNI8 and pNI9 were constructed in the same way as pNI7 except that 5'-CATCACCATCGGTCGTGCCACCGA-3' or 5'-CTTCACGGTGCCCCGGAGACG-3', respectively, was used as the reverse primer in PCR. To construct pNI10 encoding NlpI, the nlpI in the chromosomal DNA of W3110 was amplified by PCR using the forward primer containing the NcoI site (underline) 5'-TGGCCACTGAGACCTTTTTTTTTTGCG-3' and the reverse primer containing the BamHI site (underline) 5'-TGGACCCTGCCCAAATAGG-3'. The amplified DNA was digested with NcoI and BamHI and inserted into the same sites of pTrc99A. pNI12, pNI13, and pNI14 were constructed in the same way as pNI10 except that 5'-CCACGGGATCCCTTAGCTGCCCGC-3', 5'-GTCACAGGATCCCCGTCCAGGCAACG-3', or 5'-ATCTCGATGCTAGGCCCAGCAGAG-3', respectively, was used as the reverse primer containing the BamHI site (underlined) in PCR. The amplified DNA was digested with NcoI and BamHI and then inserted into the same sites of pTrc99A.

## Expression and Purification of Hexa-Histidine-Tagged NlpI Protein

_E. coli_ cells were transformed with the recombinant plasmids. The transformations were grown in L broth at 42°C for 4 h, then incubated in L broth containing 0.01% arabinose at 42°C for 4 h for expression of the fusion genes. From 400 ml of medium were collected, washed once with ice-cold 50 mM sodium phosphate buffer (pH 8.0) containing 300 mM NaCl (buffer A), suspended in 4 ml of the buffer A containing 10 mM imidazole and 1% Triton X-100, and disrupted by sonication with cooling by ice-water. The mixture was centrifuged for 15 min at 17,700 ×g to obtain a clear lysate. Four milliliters of the supernatant was mixed with 1 ml of nickel-nitrilotriacetic acid (Ni-NTA) resin and incubated with rotation for 2 h at 4°C. The mixture was loaded onto an open column. The column was washed with 4 ml each of buffer A containing 20 mM imidazole and then containing 40 mM imidazole. The bound proteins were eluted with 12 ml of buffer A containing 100 mM imidazole, and fractions of 0.4 ml were collected.

## Mass Spectrometry Analysis

An electrosonic ion trap mass spectrometer (LCQ DECA XP) ion trap mass spectrometer, Thermo Electron) coupled online with a capillary HPLC (Magic 2002, Michrom BioResources) was used to acquire MS/MS spectra. A 0.2 × 50 mm MAGICMS C18  
```