Table 1. Data collection and refinement statistics

|                    | Native      | Xe derivative    | Hg derivative   | Pt derivative     |
|--------------------|-------------|------------------|-----------------|-------------------|
|                    |             | 30-2.0 (2.07-20) |                 | 30-3.1 (3.24-3.1) |
| Unique reflections | 42,743      | 37,784           | 12,595          | 10,230            |
| Redundancy         | 3.0 (1.4)   | 9.2 (3.5)        | 3.4 (2.1)       | 2.1 (1.9)         |
| Completeness '%)   | 93.2 (58.8) | 98.2 (86.5)      | 90.1 (77.0)     | 97.3 (91.6)       |
|                    | 28.6 (2.5)  | 33.5 (2.2)       | 17.5 (3.8)      | 12.7 (1.5)        |
| %) Rmerge          | 4.4 (22.1)  | 7.0 (46.3)       | 6.0 (20.5)      | 8.1 (25.6)        |
| Refined model      |             |                  |                 |                   |
| '%) Rfree          | 26.28       |                  |                 |                   |
| Rcryst             | 20.31       |                  |                 |                   |
| rmsd values        |             |                  |                 |                   |
| Bond lengths ()    | 0.019       |                  |                 |                   |
| Bond angles        | 1.691       |                  |                 |                   |

6 proteins , we incubated each protein with a labelled DNA fragment carryE2 site 4 from the HPV 16 genome and used DNase to footprint the resulting complexes (Figure 4(a)). Both proteins protect a similar Iength of DNA from digestion by DNase I. Furthermore, the concentrations of protein required to obtain a radical footprinting revealed no significant differences in the DBD\_DNA complexes formed by the HPV 16 and HPV 6 DBDs (data not shown). Circular permutation assays a set of binding site is varied were used to investigate the effects of each DBD on DNA bending; On binding there is no significant difference in the DNA bend data clearly indicate that the DNA bend angle in the HPV 6 and HPV 16 DBD\_DNA complexes are difficulties inherent in relating DNA bend angles determined this approach to bend angles observed in protein-DNA co-crysany  certainty whether these values differ significantly from the 0 DNA bend angle of 43-515 seen in crystals of the HPV 18 and BPVI E2 complexes. Since the uncomplexed HPV 6 DBD superimposes on HPV 18 and BPVI DBDs , these data raise the possibility that the 7 A displacement observed in the structure of the HPV 16 DBD may not be replicated in the HPV 16 complex. the data   from retardation assays and footprinting experiments suggest that the HPV 6 and HPV 16 E2 DBDs bind to a consensus E2 site with similar that the protein -DNA   complexes   have similar architecture. ing using using gel

## HPV 6 and HPV 16 E2 bind preferentially to

Unlike the BPV E2 DBD, the DBD from the HPV 16 and HPV 18 E2  proteins   bind   preferentially to sites with an A:T-rich the spacer region is not contacted directly by the DBD, this difference in binding specificity must be due to the overall conformation flexibility afforded by these  base-pairs to the surrounding DNA HPV 6 E2 DBDs to E2 sites with different central base-pairs , we made a series of E2 binding sites Or CCGG in the spacer shows the affinity of the HPV 16 and Both proteins bind preferentially to an E2 site with an A:T-rich spacer (Table 2, line 1) Changing the central spacer to either ACGT or CCGG, decreases binding by a factor of around tenfold in the case of the HPV 16 DBD and by more than 1000-fold in the case of the HPV 6 The values for the HPV 16 E2 DBD differ somewhat from the 30-fold binding of this protein to E2 sites with these spacer 12 One possibility is that this could be a consequence of the different types of oligonucleotides used Whilst we have of short complementary   oligonucleotides, the previous study made use of single hairpin of double-stranded DNA. 12 regions. used pairs long oligo-

## HPV 16 and HPV 6 E2 recognise an extended binding site

An alignment of the four E2 binding sites sent within the HPV 16 LCR produces a consensus sequence that differs from that derived from the pre-