gene expression between samples was calculated through the inclusion of cDNA dilution series of a reference sample, amplified with each primer set separately. In order to account for RNA loading differences between samples; the measured gene expression was normalized against ÃŸ-actin expression. The t-test was used to assign statistical significance.

## NF-KB reporter system

was investigated luciferase reporter system in which the luciferase gene was under the control of a NF-KB were seeded in triplicate into NF-kB luciferase plasmid was transfected into them using the genejammer transfec positive control for NF-kB activity. The cells were then either exposed to DCA for varying periods of time (0-4 h) or remained untreated. After exposure, the cells were processed following the manufacturer 5 recom expression quantified with the Bright-Glo luciferase assay   system luminometer (Anthos Labtech Instruments; Salzburg, Austria). Fold increases (untreated cells were set up for each time point analysed). The t-test was used to evaluate differences between the   triplicate   readings   from treated and using using supplied

## Results

## DCA specifically switches on IKB and IL-8

RNA extracted from treated for 24 h with 300 DCA at pH 7 was analysed five different pathway specific membrane arrays. By far the most prominent   of  the DCA induced gene   expression changes were the switching on of IKB and IL-8. As IkB and IL-8 are transcriptional targets of NF-KB, this suggests activain Figure 2, using a cancer pathway array and a signal trans up-regulated 24 h after DCA treatment. Figure 2 also shows number of (mostly up-regulation) , these (and other changes noted from using

different arrays used, have been grouped into four separate groups based on the function of the genes involved. As IL-8 (29) and IKB (30) have previously been suggested to be transcriptional

| Gene                                                   | Time point   |
|--------------------------------------------------------|--------------|
| NFKB related genes                                     |              |
| Tumour necrosis factor alpha induced protein (TNFaip)  | 4-24 h       |
| Flice inhibitory protein (FLIP . Cflar, Casper)        | 24 h         |
| Cellular proteases/invasion genes                      |              |
| Plasminogen activator; Urokinase (Plau)                | 1-24 h       |
| Plasminogen activator receptor (PlauR)                 | 24 h         |
| Plasminogen activator inhibitor (Serpine 1)            | 24 h         |
| Serine protease inhibitor B5 (maspin; Serpine B5)      | 24 h         |
| Plasminogen activator inhibitor B2 (Serpine B2, PAI-2) | 24 h         |
| Growth factors/receptors                               |              |
| Melanoma growth stimulating activity (MGSA, Gro 1)     | 24 h         |
| Vascular endothelial growth   factor (VEGF)            | 4-24 h       |
| Epidermal growth receptor (EGFR) factor                | 1-24 h       |
| Adhesion molecules                                     |              |
| B-Catenin                                              | 1-24 h       |
| Integrin BI, integrin A6                               | 4-24 h       |
| E-Cadherin                                             | 1 h          |
|                                                        | 4-24 h       |
| Neural cell adhesion molecule (Ncam)                   | 1-4 h        |

This table contains a list of genes whose expression was consistently altered after exposure to DCA By consistent; we mean that the expression change was evident on both duplicate array experiments (although no real-time PCR data available). In the right hand column is the time point at which the expression change was evident The arrows in the central column reflect the relative strength of the expression changes.

2. Membrane array analysis of DCA treated 0E33 cells; 24 h post-exposure. (a) Cancer pathway arrays were used to study gene expression abnormalities A number of other genes are

<!-- image -->