1056
JOHLER ET AL.
Appl. Environ. Microbiol.

TABLE 3. Mapping of transposon insertions sites that result in white phenotype in C. sakazakii ES5
<table border=1>
<tr>
<td rowspan="2">COG functional category ${}^{a}$
</td>
<td rowspan="2">COG functional class</td>
<td colspan="2">Annotation ${}^{b,c}$</td>
</tr>
<tr>
<td>Homologue</td>
<td>Gene product</td>
</tr>
<tr>
<td colspan="4">Mutation in pigment operon</td>
</tr>
<tr>
<td>H : coenzyme transport and metabolism</td>
<td>Geranylgeranyl pyrophosphate synthase</td>
<td>crtE</td>
<td>Geranylgeranyl pyrophosphate synthase</td>
</tr>
<tr>
<td>GC: carbohydrate transport and metabolism/signal transduction mechanisms</td>
<td>Glucosyl transferases, related to UDPglucosyltransferase</td>
<td>crtX</td>
<td>Zeaxanthin glucosyl transferase</td>
</tr>
<tr>
<td>
$\mathrm{R}/\mathrm{E}$ : general function prediction only/amino acid transport and metabolism</td>
<td>Acetyltransferase/choline dehydrogenase and related flavoproteins</td>
<td>crtY</td>
<td>Lycopene cyclase</td>
</tr>
<tr>
<td>Q : secondary metabolites biosynthesis, transport and catabolism</td>
<td>Phytoene dehydrogenase and related proteins</td>
<td>crtI</td>
<td>Phytoene dehydrogenase</td>
</tr>
<tr>
<td>I: lipid transport and metabolism</td>
<td>Phytoene/squalene synthase</td>
<td>crtB</td>
<td>Phytoene synthase</td>
</tr>
<tr>
<td colspan="4">Mutation outside pigment operon</td>
</tr>
<tr>
<td rowspan="9">C : energy production and conversion</td>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$-type ATP synthase, subunit alpha</td>
<td>ESA_04012</td>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$ ATP synthase subunit alpha</td>
</tr>
<tr>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$-type ATP synthase, subunit beta</td>
<td>ESA_04006</td>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$ ATP synthase subunit beta</td>
</tr>
<tr>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$-type ATP synthase, subunit gamma</td>
<td>ESA_04007</td>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$ ATP synthase subunit gamma</td>
</tr>
<tr>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$-type ATP synthase, subunit epsilon (mitochondrial delta subunit)</td>
<td>ESA_04005</td>
<td>
$\mathrm{F}_{\mathrm{o}}\mathrm{F}_{1}$ ATP synthase subunit epsilon</td>
</tr>
<tr>
<td>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E1) component, and related enzymes</td>
<td>ESA_02622</td>
<td>sucA 2-oxoglutarate dehydrogenase E1 component</td>
</tr>
<tr>
<td>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E2) component, and related enzymes</td>
<td>ESA_02621</td>
<td>Dihydrolipoamide acetyltransferase</td>
</tr>
<tr>
<td>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E3) component, and related enzymes</td>
<td>ESA_03222</td>
<td>aceF dihydrolipoamide acetyltransferase</td>
</tr>
<tr>
<td>Malate/lactate dehydrogenases</td>
<td>ESA_03622</td>
<td>Malate dehydrogenase</td>
</tr>
<tr>
<td>Succinate dehydrogenase/fumarate reductase, flavoprotein subunit</td>
<td>ESA_02624</td>
<td>Succinate dehydrogenase flavoprotein subunit</td>
</tr>
<tr>
<td>P : inorganic ion transport and metabolism</td>
<td>
$\mathrm{Na}^{+}/\mathrm{H}^{+}$antiporter</td>
<td>ESA_03316</td>
<td>pH -dependent sodium/proton antiporter</td>
</tr>
<tr>
<td rowspan="2">T : signal transduction mechanisms</td>
<td>cAMP-binding proteins, catabolite gene activator, and regulatory subunit of cAMP-dependent protein kinases</td>
<td>ESA_04376</td>
<td>cAMP regulatory protein</td>
</tr>
<tr>
<td>DnaK suppressor protein</td>
<td>ESA_03194</td>
<td>DnaK transcriptional regulator DksA</td>
</tr>
<tr>
<td rowspan="3">S: function unknown</td>
<td rowspan="3">Uncharacterized conserved protein</td>
<td>ESA_04343 (Ent638_3811) ${}^{\text{d }}$
</td>
<td>Hypothetical protein (intracellular  growth attenuator IgA,  Enterobacter sp.  638)<span>d</span>
</td>
</tr>
<tr>
<td>ESA_03563 (ETA_03450) ${}^{d}$
</td>
<td>Hypothetical protein (YhbC-like protein, Erwinia tasmaniensis Et1/99) ${}^{d}$
</td>
</tr>
<tr>
<td>ESA_00549 (AAG53883) ${}^{d}$
</td>
<td>Hypothetical protein (sigma factor RpoS, Escherichia coli) ${}^{d}$
</td>
</tr>
</table>${ }^{a}$ NCBI clusters of orthologous groups (COG) of proteins.
${ }^{b}$ Cronobacter sakazakii ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).
${ }^{c}$ NCBI assembly ATCC BAA-894 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000783.1).
${ }^{d}$ Closest annotated homolog.
(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth ( $\mu \mathrm{max}_{\mathrm{wt}}, 0.14 ; \mu \mathrm{max}_{\Delta c r t X}, 0.22 ; \mu \max _{\Delta c r t E}, 0.24$; and $\left.\mu \max _{\Delta c r t Y}, 0.19 ; P=0.000\right)$.
Cold stress experiments were performed by growing $\Delta c r t E$, $\Delta c r t X$, and $\Delta c r t Y$ in LB and M9 medium at $10^{\circ} \mathrm{C}$ (for results in M9, see Fig. 1B). Under these conditions, no significant dif-
ferences in maximum specific growth rates were detected for $\Delta c r t Y, \Delta c r t E$, and $\Delta c r t X$ in both LB and M9 compared to those of the wild type (in LB, $\mu \max _{\mathrm{wt}}=0.04, \mu \max _{\Delta c r t X}=0.03$, $\mu \max _{\Delta c r t E}=0.03$, and $\mu \max _{\Delta c r t Y}=0.04$; in M9, $\mu \max _{\mathrm{wt}}=$ $0.01, \mu \max _{\Delta c r t X}=0.01, \mu \max _{\Delta c r t E}=0.01$, and $\mu \max _{\Delta c r t Y}=$ $0.01)$.

To evaluate growth under acidic conditions, wild-type and