TABLE 3. Mapping of transposon insertions sites that result in white phenotype in C. sakazakii ES5

| COG functional category<sup>a</sup> | COG functional class | Annotation<sup>b</sup> | Homologue | Gene product |
|----------------------------------|---------------------|------------------------|------------|--------------|
| Mutation in pigment operon       | Geranylgeranyl pyrophosphate synthase | *crtE* | Geranylgeranyl pyrophosphate synthase |
| H: coenzyme transport and metabolism | Glucosyl transferases, related to UDP-glucosyltransferase | *crnX* | Zeaxanthin glucosyl transferase |
| GC: carbohydrate transport and metabolism/signal transduction mechanisms | Acetyltransferase/choline dehydrogenase and related flavoproteins | *crnI* | Phytolene dehydrogenase |
| R/E: general function prediction only/amino acid transport and metabolism | Phytolene dehydrogenase and related proteins | *crnB* | Phytolene synthase |
| Q: secondary metabolites biosynthesis, transport and catabolism | Phytolene/dequilarine synthase | *crnF* | Phytolene synthase |
| I: lipid transport and metabolism | Phytolene/dequilarine synthase | *crnB* | Phytolene synthase |
| Mutation outside pigment operon   | F<sub>2</sub>F<sub>2</sub>-type ATP synthase, subunit alpha | *crnA* | F<sub>2</sub>F<sub>2</sub>-type ATP synthase subunit alpha |
| C: energy production and conversion | F<sub>2</sub>F<sub>2</sub>-type ATP synthase, subunit beta | *crnB* | F<sub>2</sub>F<sub>2</sub>-type ATP synthase subunit beta |
| | F<sub>2</sub>F<sub>2</sub>-type ATP synthase, subunit gamma | *crnC* | F<sub>2</sub>F<sub>2</sub>-type ATP synthase subunit gamma |
| | F<sub>2</sub>F<sub>2</sub>-type ATP synthase, subunit epsilon (mitochondrial delta subunit) | *crnD* | F<sub>2</sub>F<sub>2</sub>-type ATP synthase subunit epsilon |
| | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E1) component, and related enzymes | *crnF* | Dihydrolipoamide acetyltransferase |
| | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E2) component, and related enzymes | *crnG* | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase |
| | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E3) component, and related enzymes | *crnH* | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase |
| | Malate/lactate dehydrogenases | *crnI* | Malate dehydrogenase |
| | Succinate dehydrogenase/fumarate reductase, flavoprotein subunit | *crnJ* | Succinate dehydrogenase |
| | DnaK suppressor protein | *crnK* | Flavoprotein subunit |
| P: inorganic ion transport and metabolism | Uncharacterized conserved protein | *crnM* | DnaK suppressor protein |
| T: signal transduction mechanisms | Uncharacterized conserved protein | *crnN* | DnaK suppressor protein |
| S: function unknown | CAMP-binding proteins, catabolite gene activator, and regulatory subunit of CAMP-dependent protein kinases | *crnO* | CAMP-binding proteins, catabolite gene activator, and regulatory subunit of CAMP-dependent protein kinases |
| | DnaK suppressor protein | *crnP* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnQ* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnR* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnS* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnT* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnU* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnV* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnW* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnX* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnY* | DnaK suppressor protein |
| | Uncharacterized conserved protein | *crnZ* | DnaK suppressor protein |

<sup>a</sup> NCBI clusters of orthologous groups (COG) of proteins.
<sup>b</sup> *Cronobacter sakazakii* ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).
<sup>c</sup> NCBI assembly ATCC BAA-894 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000783.1).
<sup>d</sup> Closest annotated homolog.

(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth (μmax<sub>wt</sub> = 0.14; μmax<sub>ΔcrnE</sub> = 0.22; μmax<sub>ΔcrnX</sub> = 0.24; and μmax<sub>ΔcrnY</sub> = 0.19; P = 0.000).

Cold stress experiments were performed by growing ΔcrnE, ΔcrnX, and ΔcrnY in LB and M9 medium at 10°C (for results in M9, see Fig. 1B). Under these conditions, no significant differences in maximum specific growth rates were detected for ΔcrnY, ΔcrnE, and ΔcrnX in both LB and M9 compared to those of the wild type (in LB, μmax<sub>wt</sub> = 0.04, μmax<sub>ΔcrnX</sub> = 0.03, μmax<sub>ΔcrnE</sub> = 0.03, and μmax<sub>ΔcrnY</sub> = 0.04; in M9, μmax<sub>wt</sub> = 0.01, μmax<sub>ΔcrnX</sub> = 0.01, μmax<sub>ΔcrnE</sub> = 0.01, and μmax<sub>ΔcrnY</sub> = 0.01).

To evaluate growth under acidic conditions, wild-type and